docker/polygon_zkevm/polygon_zkevm.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+. "$(realpath $(dirname $0))/../scripts/common.sh"
+
+ROOT="$(get_root_dir)"
+
+# Deployer's address & private key.
+ADDRESS=0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266
+PRIVKEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
+
+CACHE_DIR=$(obtain_cache_dir $0)
+POLYGON_ZKEVM="${CACHE_DIR}/polygon_zkevm"
+DEPLOYMENT_DIR="${CACHE_DIR}/deployment"
+DEPLOYMENT_CONFIG_FILE="${CACHE_DIR}/polygon-zkevm-local.json"
+KEYFILE="${CACHE_DIR}/${ADDRESS}.json"
+
+ensure_keyfile_exists ${PRIVKEY} ${KEYFILE}
+echo Beamer deployer\'s keyfile: ${KEYFILE}
+
+down() {
+    echo "Shutting down the end-to-end environment"
+    if [ ! -d ${POLYGON_ZKEVM} ]; then
+        configure_repo
+    fi
+    make -C ${POLYGON_ZKEVM}/test stop
+}
+
+configure_repo() {
+
+    repo="https://github.com/0xPolygonHermez/zkevm-node.git"
+    commit_id="ce27fdd2f45e93bce47b6c78f644d583a668599b"
+    git clone --no-checkout ${repo} ${POLYGON_ZKEVM}
+    cd ${POLYGON_ZKEVM}
+    git checkout ${commit_id}
+}
+
+create_deployment_config_file () {
+    if [ ! -d ${POLYGON_ZKEVM} ]; then
+        configure_repo
+        make -C ${POLYGON_ZKEVM} build-docker
+    fi
+    ADDRESS=$(bridge_address)
```

Comment:
Would be nice to avoid the same variable name as the global one, e.g `bridge_address` could work...